### ISSUE TITLE
Add some indications on the intents on the feedback page

### ISSUE THREAD

**Issue by callicles** [gitlinks/gitrank-web#121]
The feedback form should give some insight in what is expected from the user : ie meaningful opinion, not bug report

**Comment by callicles** [comment]
@sheymann Any suggestions how we could do that ?

**Comment by sheymann** [comment]
@callicles Writing some text? Splitting the textarea into a few textareas with one question as title and a sentence as subtitle to explain? I like the airbnb style of giving feedback to hosts with open questions.

**Comment by callicles** [comment]
I agree with the adding of a subtile to help the user, however isn't creating multiple field risk turning the user away ?

**Comment by sheymann** [comment]
3 or 4 (optional) fields to guide the user
